src/pages/HomePage/HomePage.tsx

```diff
@@ -94,7 +94,7 @@ function HomePage() {
       <Container className={homePageSyles.container}>
```

Comment:
Why do we have only spaces as a change for this file?

Reply:
For some reason my IDE adds a new line at the end of each file🤔